src/content/Interoperability.js

```diff
@@ -219,7 +227,11 @@ exists(f)`}
                 <SubSection name="Summary">
 
                     <p>
-                        The table below gives an overview of the syntax:
+                        The table below gives an overview of the syntax.
+                    </p>
+
+                    <p>
+                        Note: the return types and effects must always be specifed but are omitted for a simpler overview.
```

Comment:
I opted for a simple note above the overview table for imports.

Reply:
👍